_posts/2018-03-10-meetup-march.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Meetup Summary (March, 2018"
+---
+
+The March Monthly meetup was held at Adlife Plaza. Members started
+streaming in from 1400Hrs.
+
+Black Panther is a recent film based on a Marvel Character by the
+same name. It is basically about a country in Africa, that was
+successfully able to avoid colonization and has been hiding its
+technological prowess from the world.
+
+The movie has been lauded by critics and audiences all over. Some
```

Comment:
Make this paragraph part of the previous paragraph. It makes more sense that way